ui/raidboss/data/06-ew/raid/p12s.txt

```diff
@@ -4,3 +4,169 @@ hideall "--Reset--"
+100.0 "Glaukopis" sync / 1[56]:[^:]*:Athena:82FD:/
+116.1 "Superchain Theory I" sync / 1[56]:[^:]*:Athena:82DA:/
+123.2 "Engravement of Souls" sync / 1[56]:[^:]*:Athena:8305:/
+128.2 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+134.3 "Polarized Ray" sync / 1[56]:[^:]*:Athena:8307:/
+135.2 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+135.5 "Astral/Umbral Impact" sync / 1[56]:[^:]*:Athena:830[89]:/ window 3,0.5
+140.2 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+142.2 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 1,0.5
+143.5 "Astral/Umbral Glow" sync / 1[56]:[^:]*:Athena:830[CD]:/ window 3,0.5
+145.2 "Theos's Holy" sync / 1[56]:[^:]*:Athena:8306:/ window 3,0
+146.4 "Astral/Umbral Advance" sync / 1[56]:[^:]*:Anthropos:830[FE]:/ window 3,0.5
+157.4 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/
+160.1 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/
+162.7 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/
+171.1 "Apo/Peridialogos" sync / 1[56]:[^:]*:Athena:830[01]:/
+172.1 "Dialogos" sync / 1[56]:[^:]*:Athena:8302:/
+178.8 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/
+190.0 "--sync--" sync / 1[56]:[^:]*:Athena:8315:/
+195.3 "Paradeigma" sync / 1[56]:[^:]*:Athena:82ED:/
+202.4 "Engravement of Souls" sync / 1[56]:[^:]*:Athena:8305:/
+212.5 "Shock" sync / 1[56]:[^:]*:Athena:8312:/
+213.7 "Searing Radiance/Shadowsear" sync / 1[56]:[^:]*:Anthropos:82F[12]:/ window 3,0.5
+220.5 "Theos's Saltire/Cross" sync / 1[56]:[^:]*:Athena:830[AB]:/ window 3,0.5
+220.8 "Astral/Umbral Glow" sync / 1[56]:[^:]*:Athena:830[CD]:/ window 3,0.5
+221.6 "White Flame" #sync / 1[56]:[^:]*:Anthropos:82F0:/
+223.8 "Astral/Umbral Advance" sync / 1[56]:[^:]*:Anthropos:830[EF]:/ window 3,0.5
+229.8 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/
+242.1 "Glaukopis" sync / 1[56]:[^:]*:Athena:82FC:/
+245.2 "Glaukopis" sync / 1[56]:[^:]*:Athena:82FD:/
+254.3 "--sync--" sync / 1[56]:[^:]*:Athena:8315:/
+259.9 "--sync--" sync / 1[56]:[^:]*:Athena:82F3:/
+267.3 "Ultima Blade" sync / 1[56]:[^:]*:Athena:82F4:/
+280.0 "--untargetable--"
+# Laser/Cleave order is random here so we can only really timeline the Palladion dashes.
+290.6 "Palladion 1" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+293.7 "Palladion 2" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+296.8 "Palladion 3" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+299.9 "Palladion 4" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+303.0 "Palladion 5" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+306.1 "Palladion 6" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+309.2 "Palladion 7" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+312.4 "Palladion 8" sync / 1[56]:[^:]*:Athena:82F6:/ window 1.5,0.5
+317.5 "--sync--" sync / 1[56]:[^:]*:Athena:82F8:/ window 1.5,0.5
+321.5 "Palladion Floor Drop" sync / 1[56]:[^:]*:Athena:82F9:/ window 1.5,0.5
```

Comment:
```suggestion
317.5 "--sync--" sync / 1[56]:[^:]*:Athena:82F8:/
321.5 "Palladion (Floor Drop)" sync / 1[56]:[^:]*:Athena:82F9:/
```

These shouldn't need windows, I don't think?

Reply:
Was likely leftover code when I was experimenting with windows when testing the timeline against as many logs as I could get my hands on, apologies.